src/video/v9990/V9990.cc

```diff
@@ -63,6 +64,16 @@ V9990::V9990(const DeviceConfig& config)
 	, v9990PalDebug(*this)
 	, irq(getMotherBoard(), getName() + ".IRQ")
 	, display(getReactor().getDisplay())
+	, invalidRegisterReadCallbackSetting(getCommandController(),
+		"v9990_invalid_register_read_callback",
+		"Tcl proc to call when a write-only register was read from. "
+		"Input: register number (0x0-0x63)", {})
+	, invalidRegisterReadCallback(invalidRegisterReadCallbackSetting)
+	, invalidRegisterWriteCallbackSetting(getCommandController(),
+		"v9990_invalid_register_write_callback",
+		"Tcl proc to call when a read-only register was written to. "
+		"Input: register number (0x0-0x63)", {})
+	, invalidRegisterWriteCallback(invalidRegisterWriteCallbackSetting)
```

Comment:
While reviewing your previous commit I noticed that the `TclCallback` class offers the possibility to internally store a StringSetting. With that your code can be simplified to:
```
, invalidRegisterWriteCallback(getCommandController(),
		"v9990_invalid_register_write_callback",
		"Tcl proc to call when a read-only register was written to. "
		"Input: register number (0-63)",
                {}, Setting::SAVE)
```
(And same for read).
And then you can remove the two `StringSetting` objects from the V9990.hh file.